.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: python
-python: 2.7
+python: 
+    - 2.7
+    - 3.5
 addons:
   firefox: 47.0.1
 env:
-- TOXENV=tests DISPLAY=:99.0
+- TOXENV=tests DISPLAY=:99.0 FILES="tests/*.py tests/webdriver/*.py"
```

Comment:
Move `DISPLAY=:99.0` to be a [global variable](https://docs.travis-ci.com/user/environment-variables#Global-Variables).